lntemp/rpc/chain_kit.go

```diff
@@ -0,0 +1,62 @@
+package rpc
+
+import (
+	"context"
+
+	"github.com/lightningnetwork/lnd/lnrpc/chainrpc"
+)
+
+// =====================
+// ChainKitClient related RPCs.
+// =====================
+
+// GetBestBlock makes a RPC call to chain kit client's GetBestBlock and asserts.
+func (h *HarnessRPC) GetBestBlock(
+	req *chainrpc.GetBestBlockRequest) *chainrpc.GetBestBlockResponse {
+
+	if req == nil {
+		req = &chainrpc.GetBestBlockRequest{}
+	}
+
+	ctxt, cancel := context.WithTimeout(h.runCtx, DefaultTimeout)
+	defer cancel()
+
+	resp, err := h.ChainKit.GetBestBlock(ctxt, req)
+	h.NoError(err, "GetBestBlock")
+
+	return resp
+}
+
+// GetBlock makes a RPC call to chain kit client's GetBlock and asserts.
+func (h *HarnessRPC) GetBlock(
+	req *chainrpc.GetBlockRequest) *chainrpc.GetBlockResponse {
+
+	if req == nil {
+		req = &chainrpc.GetBlockRequest{}
+	}
+
+	ctxt, cancel := context.WithTimeout(h.runCtx, DefaultTimeout)
+	defer cancel()
+
+	resp, err := h.ChainKit.GetBlock(ctxt, req)
+	h.NoError(err, "GetBlock")
+
+	return resp
+}
+
+// GetBlockHash makes a RPC call to chain kit client's GetBlockHash and asserts.
+func (h *HarnessRPC) GetBlockHash(
```

Comment:
Nice! I forgot to add a comment about adding those helper function in the original PR...

Reply:
Maybe I should put some notes about how to add new RPC clients/endpoints in the `lntest`'s readme...